diff --git a/n.fish b/n.fish
index c990208..a9d4466 100644
--- a/n.fish
+++ b/n.fish
@@ -29,7 +29,7 @@ function n --wraps nnn --description 'support nnn quit and change directory'

     # The command function allows one to alias this function to `nnn` without
     # making an infinitely recursive alias
-    command nnn $argv
+    nnn $argv

     if test -e $NNN_TMPFILE
         source $NNN_TMPFILE
